src/EmbeddedWebserver.h

```diff
@@ -267,6 +272,26 @@ void serverSetup() {
         request->redirect("/");
     });
 
+#if (ONLYPIDSCALE == 1 || BREWMODE == 2)
+    server.on("/toggleTare", HTTP_POST, [](AsyncWebServerRequest *request) {
```

Comment:
When do I have to manually tare?

Reply:
Manual tare is not really needed. It is a remainder of testing and of the idea of always weighing.
The scale will auto tare in the process of brewing.